platform/commonUI/general/res/sass/controls/_indicators.scss

```diff
@@ -120,6 +120,16 @@ mct-indicators mct-include {
                 top: 100%;
                 @include triangle('down', $size: 4px, $ratio: 1, $color: $hbg);
             }
+            
+            &:after {
+                // Hit area
+                $p: -7px;
+                content: '';
+                display: block;
+                position: absolute;
+                top: $p; right: $p; bottom: $p; left: $p;
```

Comment:
Hi @larkin 
I think, having the negative z-index can be a trouble here.
This will make this element having the least priority in the entire application.

Is there any specific reason for not giving it a positive value?